backend/src/test/java/io/debezium/configserver/ListPostgresConnectorIT.java

```diff
@@ -0,0 +1,72 @@
+package io.debezium.configserver;
+
+import io.debezium.configserver.rest.ConnectorResource;
+import io.debezium.configserver.util.Infrastructure;
+import io.debezium.configserver.util.PostgresInfrastructureTestProfile;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.Matchers.is;
+
+@QuarkusTest
+@TestProfile(PostgresInfrastructureTestProfile.class)
+public class ListPostgresConnectorIT {
+
+    @BeforeEach
+    public void resetRunningConnectors() {
+        try {
+            Infrastructure.getDebeziumContainer().deleteAllConnectors();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @Test
+    public void testListConnectorsEndpointEmpty() {
+        given().when().get(ConnectorResource.API_PREFIX + ConnectorResource.LIST_CONNECTORS_ENDPOINT, "1")
+                .then().log().all()
+                .statusCode(200)
+                .body("size()", is(0));
+    }
+
+    @Test
+    public void testListConnectorsEndpoint() {
+        final var runningConnectorName = "list-connectors-postgres-connector";
+        final var pausedConnectorName = "list-connectors-postgres-connector-paused";
+        try {
+            Infrastructure.getDebeziumContainer().registerConnector(
```

Comment:
Very nice with the API extension for TC 👍

Reply:
thank you